docs/options.md

```diff
@@ -539,8 +573,7 @@ CIBW_DEPENDENCY_VERSIONS: ./constraints.txt
 
 Shell command to run tests after the build. The wheel will be installed
 automatically and available for import from the tests. To ensure the wheel is
-imported by your tests (instead of your source copy), tests are run from a
-different directory. Use the placeholders `{project}` and `{package}` when
+imported by your tests (instead of your source copy), **tests are not run from your project directory**. Use the placeholders `{project}` and `{package}` when
```

Comment:
Line breaks are slightly inconsistent here, now, if you care about that.

Reply:
I've long been a hard-wrapper, but lately I've been wondering about stripping out the manual line breaks, and turning on Soft Wrap in my editor. It's kinda tedious to have to manually reformat all the time. And it would be a bit nicer to have diff see each paragraph as a single instance. What do you think?

Reply:
I don't really mind either way. I'm never a fan of imposed line limits (though @henryiii is, I believe), so soft-wrapping is perfect for me.